pages/[username]/index.tsx

```diff
@@ -14,12 +19,64 @@ type UserGalleryProps = MetaTagProps & {
   username: string;
 };
 
+type GalleryViewEmitterProps = {
+  queryRef: UsernameGalleryViewQueryFragment$key;
+};
+
+function GalleryViewEmitter({ queryRef }: GalleryViewEmitterProps) {
```

Comment:
all this component is emit the mutation to our backend that a gallery has been viewed.

i made it its own component since it has data requirements and i want to put a suspense boundary around. we don't want this little thing to cause a full screen loader